### PR TITLE
chore(DirListing): hide hidden files

### LINKED ISSUE #135: Hide certain files
In particular, don't show `.ipynb_checkpoints`, maybe other hidden files.

### PR DESCRIPTION
**Pull request by alexandercbooth**
Closes #135.

### CODE REVIEW

File: packages/commuter-directory-listing/src/DirectoryListing.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ const DirectoryListing = props => {
             <Table.Row />
           </Table.Header>
           <Table.Body>
-            {props.contents.map((row, index) => {
+            {props.contents.filter((z) => {
+               return z.name[0] !== '.';
```
Comment (alexandercbooth) [inline packages/commuter-directory-listing/src/DirectoryListing.js]: Nice, I hadn't even thought of using that!

**Comment by alexandercbooth** [comment]
I am _super_ excited about commuter, by the way (first time I've taken it for a spin); I feel like possibilities are endless.

**Comment by rgbkrk** [comment]
This works for now -- in the future it would be best to have the initial file listing be trimmed in the reducer (instead of this component) or to have the server return less (that's not specced though).

**Comment by alexandercbooth** [comment]
Sorry about that! Just submitted #147 to move this to the reducer.

**Pull request merged**